src/schema/rules/tabular_metadata.yaml

```diff
@@ -7,8 +7,8 @@ scans:
   - .json
   entities:
     subject: required
-    session: optional # session is required if session is present in the dataset.
-sessions: # This file may only exist if session is present in the dataset.
+    session: optional  # session is required if session is present in the dataset.
+sessions:  # This file may only exist if session is present in the dataset.
```

Comment:
completely unrelated to this PR thus better be dropped, in particular there is no uniforming

```shell
(git)lena:~/proj/bids/bids-specification[master]src/schema
$> git grep '[^ ] #'
objects/metadata.yaml:    `"Siemens WIP ### version #.##,"` or
objects/metadata.yaml:    `"Rabbit anti-Human HTR5A Polyclonal Antibody, Invitrogen, Catalog # PA1-2453"`).
objects/metadata.yaml:    `"Goat anti-Mouse IgM Secondary Antibody, Invitrogen, Catalog # 31172"`).
rules/tabular_metadata.yaml:    session: optional # session is required if session is present in the dataset.
rules/tabular_metadata.yaml:sessions: # This file may only exist if session is present in the dataset.

$> git grep '[^ ]  #'
objects/extensions.yaml:  name: AVE  # not sure what ave stands for
rules/checks/events.yaml:    level: warning  # could be an error with the proper selectors, I think
rules/datatypes/anat.yaml:  - T2star  # deprecated
rules/datatypes/anat.yaml:  - FLASH  # deprecated
rules/datatypes/anat.yaml:  - PD  # deprecated
rules/datatypes/func.yaml:  - phase  # deprecated
rules/datatypes/meg.yaml:  - /  # corresponds to BTi/4D data
```

Reply:
I fail to see how those `grep` calls show a lack of uniformity. The `object/metadata.yaml` uses are within string literals and do not refer to end-of-line comments, while the other two in the first `grep` are the ones modified in this PR.

Reply:
gy -- I was blind seeing the hit in `rules/tabular_metadata.yaml` which is fixed here...

overall, if to unify -- good -- but a separate PR and ideally some way to "enforce" that in the future or such non-uniformities will creep in. ... apparently yamllint already does that but issues just a warning by default

```shell
$> yamllint src/schema/rules/tabular_metadata.yaml
src/schema/rules/tabular_metadata.yaml
  10:23     warning  too few spaces before comment  (comments)
  11:11     warning  too few spaces before comment  (comments)
```
so it is just a matter of adjusting
```
diff --git a/.yamllint.yml b/.yamllint.yml
index c18c17fe..19ff3fb8 100644
--- a/.yamllint.yml
+++ b/.yamllint.yml
@@ -6,3 +6,5 @@ rules:
   indentation:
     # See https://github.com/yaml/pyyaml/issues/545 for why
     indent-sequences: false
+  comments:
+    level: error
```
and while at that address the 
```
src/schema/rules/checks/func.yaml
  1:1       warning  missing document start "---"  (document-start)
```

Reply:
FWIW, submitted a dedicated PR for this: https://github.com/bids-standard/bids-specification/pull/1117